docs/Makefile

```diff
@@ -16,7 +16,7 @@ configure: clean
 
 .PHONY: kmdo
 kmdo:
-	kmdo src
+	kmdo src || exit 0
```

Comment:
Please split this into its own commit w.g. "docs:build: osx related work-around"

Reply:
Moved into a related OSX commit on #7.